instrumentation/slowrendering/src/main/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderingInstrumentation.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.slowrendering;
+
+import android.app.Application;
+import android.os.Build;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import io.opentelemetry.android.OpenTelemetryRum;
+import io.opentelemetry.android.common.RumConstants;
+import io.opentelemetry.android.instrumentation.AndroidInstrumentation;
+import java.time.Duration;
+
+/** Entrypoint for installing the slow rendering detection instrumentation. */
+public final class SlowRenderingInstrumentation implements AndroidInstrumentation {
+
+    Duration slowRenderingDetectionPollInterval = Duration.ofSeconds(1);
+
+    /**
+     * Configures the rate at which frame render durations are polled.
+     *
+     * @param interval The period that should be used for polling.
+     * @return {@code this}
+     */
+    public SlowRenderingInstrumentation setSlowRenderingDetectionPollInterval(Duration interval) {
+        if (interval.toMillis() <= 0) {
+            Log.e(
+                    RumConstants.OTEL_RUM_LOG_TAG,
+                    "Invalid slowRenderingDetectionPollInterval: "
+                            + interval
+                            + "; must be positive");
+            return this;
+        }
+        this.slowRenderingDetectionPollInterval = interval;
+        return this;
+    }
+
+    @Override
+    public void install(
```

Comment:
Perhaps we can add a `@RequiresApi` or `@ChecksSdkIntAtLeast` annotation here or to the class itself to flag attempted usage in a lower API level?1020.

Reply:
I didn't think of it because this method (at least for now) is meant to be called by the agent and not directly by our users, although I guess it's not impossible that someone would want to call it directly, so I guess it's probably better to make sure. I'll add the changes.

Reply:
I've made the changes.